diff --git a/node_modules/html-webpack-plugin/lib/compiler.js b/node_modules/html-webpack-plugin/lib/compiler.js
index 8960360..bf9a928 100644
--- a/node_modules/html-webpack-plugin/lib/compiler.js
+++ b/node_modules/html-webpack-plugin/lib/compiler.js
@@ -337,14 +337,15 @@ function isChildCompilerCacheOutdated (mainCompilation, childCompiler) {
   }
   // Check if any dependent file was changed after the last compilation
   const fileTimestamps = mainCompilation.fileTimestamps;
-  const isCacheOutOfDate = childCompiler.fileDependencies.some((fileDependency) => {
-    const timestamp = fileTimestamps.get(fileDependency);
-    // If the timestamp is not known the file is new
-    // If the timestamp is larger then the file has changed
-    // Otherwise the file is still the same
-    return !timestamp || timestamp > childCompiler.compilationStartedTimestamp;
-  });
-  return isCacheOutOfDate;
+  return false;
+  // const isCacheOutOfDate = childCompiler.fileDependencies.some((fileDependency) => {
+  //   const timestamp = fileTimestamps.get(fileDependency);
+  //   // If the timestamp is not known the file is new
+  //   // If the timestamp is larger then the file has changed
+  //   // Otherwise the file is still the same
+  //   return !timestamp || timestamp > childCompiler.compilationStartedTimestamp;
+  // });
+  // return isCacheOutOfDate;
 }
 
 module.exports = {
